keyboard-navigation.js.tid

```diff
@@ -167,17 +193,12 @@ document.onkeyup = function(e) {
 	tiddlers = document.getElementsByClassName("tc-tiddler-frame");
 	if (tiddlers.length == 0) return;
 
-	var tiddlerIgnoreKeyboardLayout = $tw.wiki.getTiddler("$:/config/IgnoreKeyboardLayout");
-	ignoreKeyboardLayout = (tiddlerIgnoreKeyboardLayout.fields.text.trim() == "1");
-
 	var tiddlerCycle = $tw.wiki.getTiddler("$:/config/Cycle");
 	cycle = (tiddlerCycle.fields.text.trim() == "1");
 
-	var j_released = ignoreKeyboardLayout ? (e.code == "KeyJ") : (e.key == "j");
-	var k_released = ignoreKeyboardLayout ? (e.code == "KeyK") : (e.key == "k");
-	var c_released = ignoreKeyboardLayout ? (e.code == "KeyC") : (e.key == "c");
-	// e: check for ArrowUp because AltGr+E is ArrowUp in Neo2, AdNW, KOY and other keyboard layouts
-	var e_released = ignoreKeyboardLayout ? (e.code == "KeyE" && e.key != "ArrowUp") : (e.key == "e");
```

Comment:
Changing this is really bad for users of Neo and other modern keyboard layouts. In my layout, `d` is mapped to `e` and `AltGr` + `d` (=`e` in my layout) is mapped to the down arrow. Now, this will be interpreted as `e` (edit tiddler) although I just want to scroll down.